src/shared/constants.js

```diff
@@ -1,2 +1,3 @@
 export const MESSAGE_JOB_REQUEST = 'requestJob';
 export const MESSAGE_JOB_STATUS = 'jobStatus';
+export const METRIC_DENOMINATOR = 2.54;
```

Comment:
2.54 is the amount of inches in one centimer, not sure the name is releavant.

Reply:
ok I renamed it